website/source/api/operator.html.md

```diff
@@ -420,5 +420,5 @@ The table below shows this endpoint's support for
 ```
 
 - `PreemptionConfig` `(PreemptionConfig)` - Options to enable preemption for various schedulers.
- - `SystemSchedulerEnabled`(bool:true) - Specifies whether preemption for system jobs is enabled. Note that
-         if this is set to true, then system jobs can preempt any other jobs.
\ No newline at end of file
+ - `SystemSchedulerEnabled` (`bool`: `true`) - Specifies whether preemption for system jobs is enabled. Note that
```

Comment:
I guess everything else is just doing ticks around the entire parenthesis. Was thinking about how the job spec does it.